contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -581,10 +581,10 @@ <h3 id="capability">Add a Capability to this Contribution</h3>
                     <div name="add-env-var-div" class="form-group">
                         <label class="required control-label col-sm-2">Environment Variable(s)</label>
                         <div class="col-sm-10" id="key-val">
-                            <input class="form-control" name="environmentVariables_key" placeholder="Key"
-                                   id="environmentVariables_key" type="text">
-                            <input class="form-control" name="environmentVariables_value" placeholder="Value"
-                                   id="environmentVariables_value" type="password">
+                            <input class="form-control" name="environmentVariables_key_0" placeholder="Key"
```

Comment:
Similarly, please add some comments about how the environment variable ID indexes are searched and updated. Thanks.